### PR TITLE
Add client_type field to logging fields.

### PR DESCRIPTION
**Pull request by milkypostman**

### CODE REVIEW

**Comment by milkypostman** [comment]
@jiayliu @tkchin @chuckhays

File: src/app_engine/bigquery/enums.json [hunk 1]
```diff
@@ -18,5 +19,13 @@
       "ROOM_ID": "room_id",
       "EVENT_TIME_MS": "event_time_ms"
     }
+  },
+
+  "ClientType": {
+    "UNKNOWN": 0,
+    "JS": 1,
+    "DESKTOP": 2,
```
Comment (jiayliu) [inline src/app_engine/bigquery/enums.json]: What's desktop?
Do we need to differentiate Chrome vs Chrome App?
What about linux/mac/win?

**Comment by milkypostman** [comment]
Updated on top of #130 . ptal.

**Comment by jiayliu** [comment]
lgtm

File: src/app_engine/bigquery/enums.json (hunk 1 above)
Comment (tkchin) [inline src/app_engine/bigquery/enums.json]: On a similar note do we care about iPad vs iPhone vs Android tablet?

**Comment by tkchin** [comment]
lgtm

**Pull request merged**